lib/pbench/server/database/models/datasets.py

```diff
@@ -612,6 +612,13 @@ class Metadata(Database.Base):
     METALOG = "metalog"
     NATIVE_KEYS = [GLOBAL, METALOG, SERVER, USER]
 
+    # BENCHMARK provides a simple identification of the native benchmark where
+    # one can be identified. At the simplest for Pbench Agent tarballs this is
+    # just "dataset.metalog.pbench.script" but will be expanded in the future
+    # for example to identify a "pbench-user-benchmark -- fio" as "fio".
+    SERVER_BENCHMARK = "server.benchmark"
+    SERVER_BENCHMARK_UNKNOWN = "unknown"
```

Comment:
It probably would be good to have a reference here to possible values for this key (i.e., to `BenchmarkName` or `endpoints["visualization"]["benchmarks"]`?).

Reply:
Those would constitute *examples*, but I already added `fio` as an example. Maybe eventually we'll support visualization of all possible benchmarks, but right now either of those references would be incomplete.